python/src/file.rs

```diff
@@ -113,6 +115,51 @@ impl LanceColumnMetadata {
     }
 }
 
+#[pyclass(get_all)]
+#[derive(Clone, Debug, Serialize)]
+pub struct LanceFileStatistics {
+    columns: Vec<LanceColumnStatistics>,
+}
+
+#[pymethods]
+impl LanceFileStatistics {
+    fn __repr__(&self) -> String {
+        let column_reprs: Vec<String> = self.columns.iter().map(|col| col.__repr__()).collect();
+        format!("LanceFileStatistics(columns=[{}])", column_reprs.join(", "))
+    }
+}
+
+#[pyclass(get_all)]
+#[derive(Clone, Debug, Serialize)]
+pub struct LanceColumnStatistics {
+    num_pages: usize,
+    size_bytes: u64,
+}
```

Comment:
Since `LanceColumnStatistics` and `LanceFileStatistics` are exposed directly to the user (being re-exported in file.py) we should add docstrings.  You can copy the ones I suggested in the rust lib.

Reply:
fixed, thanks.